sdks/desktop/Makefile

```diff
@@ -0,0 +1,15 @@
+TOP=$(realpath $(CURDIR)/../..)
```

Comment:
We should have these targets in `sdks/android`

Reply:
We're going to have duplicated code if we have the desktop side of the debugger test in both the android and the ios folders. By keeping it in desktop, we factor out what would be duplicated code.

Reply:
Makes sense, let's keep it there then.